www/js/splash/startprefs.ts

```diff
@@ -0,0 +1,121 @@
+import { storageGet, storageSet } from '../plugin/storage';
+import { logInfo, logDebug, displayErrorMsg } from '../plugin/logger';
+
+type StartPrefs = {
+  CONSENTED_EVENT: string,
+  INTRO_DONE_EVENT: string,
+}
+
+export const startPrefs: StartPrefs = {
+  CONSENTED_EVENT: "data_collection_consented",
+  INTRO_DONE_EVENT: "intro_done",
+};
```

Comment:
At the end of this, I actually don't think you will need this `startPrefs` variable at all.
Because they're just for emitting angular events and we should be getting rid of those

Reply:
removed in [a7a75d1](https://github.com/e-mission/e-mission-phone/pull/1072/commits/a7a75d1dde7f5621074a514b9ec9c0a71235e732)